tests/conftest.py

```
@@ -12,7 +12,87 @@ def pytest_configure(config):
+    from rest_framework.test import APIClient
+
+    from bf_opencore.tests.factories import make_user
+
+    user = make_user()
+    api_client = APIClient()
+    api_client.force_authenticate(user=user)
+    return api_client
+
+
+@pytest.fixture
+def admin_client(db):
```

Ruff (0.15.1)

[warning] Unused function argument: `db`

(ARG001)